File: src/main/java/com/example/healthylifestyleapp/FitnessDevice.java
package com.example.healthylifestyleapp;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

public class FitnessDevice extends AppCompatActivity implements View.OnClickListener {
    ImageView imgBackSpaceFitness;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fitness_device);
        imgBackSpaceFitness=findViewById(R.id.imgBackSpaceFitness);
        imgBackSpaceFitness.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId())
        {
            case R.id.imgBackSpaceFitness:
                Intent fitnessIntent = new Intent(this, UserProfileActivity.class);
                startActivity(fitnessIntent);
                break;
        }
    }
}

File: app/src/main/java/com/example/healthylifestyleapp/ui/activities/FitnessDevice.kt
package com.example.healthylifestyleapp.ui.activities

import android.content.Intent
import android.os.Bundle
import android.view.View

import androidx.appcompat.app.AppCompatActivity
import com.example.healthylifestyleapp.R
import kotlinx.android.synthetic.main.activity_fitness_device.*

class FitnessDevice : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fitness_device)
        imgBackSpaceFitness.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.imgBackSpaceFitness -> {
                val fitnessIntent = Intent(this, UserProfileActivity::class.java)
                startActivity(fitnessIntent)
            }
        }
    }
}
